src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -376,6 +376,7 @@
 
 (defn overview
   [id]
+  (rf/dispatch [:chat.ui/fetch-community id])
```

Comment:
hi @yqrashawn, Thank you very much for your PR.

Can we create an new event, `:navigate-to-community` and add this dispatch there?